3_harmonize.R

```diff
@@ -66,10 +63,10 @@ p3_targets_list <- list(
     p3_wqp_data_aoi_clean_param,
     {
       # Decide which function to use
-      fxn_to_use <- p3_wqp_data_aoi_clean_grp %>%
+      fxn_to_use <- p3_wqp_param_cleaning_info %>%
+        filter(parameter == unique(p3_wqp_data_aoi_clean_grp$parameter)) %>%
         pull(cleaning_fxn) %>%
-        names() %>% 
-        unique()
+        .[[1]]
```

Comment:
Looks good and this works as expected. 

I'm not sure if this is good overall advice or not, but I tend to try to use `{}` whenever I'm accessing the `.` outside of a function. In this case, it doesn't matter but sometimes scoping the accessor method by doing `x %>% {.[[1]]}` avoids surprising issues. 

Up to you if you want to make that change in this case but figured I'd mention it. 

One example of a surprising result in a more complicated case is this

```r
list(t = 4, y = 3) %>% {c(.[[1]], .[[2]])}
[1] 4 3

list(t = 4, y = 3) %>% c(.[[1]], .[[2]])

$t
[1] 4

$y
[1] 3

[[3]]
[1] 4

[[4]]
[1] 3
```

Reply:
Thanks for pointing to this reprex - I think you provide good reason for adding the `{}`.